### DISCUSSION TITLE
Issues setting up the trigger

### DISCUSSION THREAD

**Discussion by jguipi** [demmings/cachefinance#8] [Q&A] [answered]
Hi, 
I'm not able to set up the trigger, when I run the function `CacheFinanceBoot` nothing happen and the price of the function `CACHEFINANCE` seems to never update. 

Can you share one of your personal spreadsheet as an example or provide a video that goes through the setup process ?

Thanks

**Comment by demmings** [comment, marked as answer]
Hey jguipi,
I probably need to update and simplify the Trigger program and documentation (first version).  I do testing here:

https://docs.google.com/spreadsheets/d/1zk7GIJ7ys4hBhiZwrLemqIGf-lH7eiLcJWKw6IkdAY4/edit?usp=sharing

There is a lot more setup to this compared to using CACHEFINANCE(), but there are some advtantages to running as a trigger.
The important things are:
1)  Setup the NAMED RANGE:  'CACHEFINANCE' that contains the table of setup parameters.
The ranges specified in the table are strings, like "Stocks!C3:C133"
2)  Set up the stock symbols consecutively in a column
3) Set up a column of default data that comes from googlefinance
4) once configured, start up the triggers by activating  "=CacheFinanceBoot()".  I just change the parameter to a different value (it is just ignored) - but it forces google to run when there is a change.
5)  It takes a few minutes before the triggers load up.
6) When adding symbols AFTER the triggers are running, nothing is updated until the trigger run again.
7) Check if any triggers are created in App Script (the alarm clock).

It needs some streamlining work.  It works for me, but cumbersome to get going initially.
The primary reason I use is that I found using CACHEFINANCE() with a few hundred stock symbols would start to cause LOADING errors (30 seconds max for custom functions) - when a single trigger could update dozens of prices and it has 300 seconds before timing out.  If it comes close to running out of time, it exists the function and restarts where it left off.

**Reply by jguipi** [reply]
nice ! 
I'll give it a try. 
Just sent a request for the spreadsheet :) 

thx

**Reply by jguipi** [reply]
I was able to make it work, thx !
